### ISSUE TITLE
About MSD testing sets

### ISSUE THREAD

**Issue by jsadu826** [Luffy03/Large-Scale-Medical#2]
Dear author, I'd like to know whether the results of MSD tasks were obtained from the official testing sets or custom testing sets split from the official training sets?

**Comment by Luffy03** [comment]
Hi, the results are from the splits of training sets, which are generated from 'split_final.json' of nnUNet.

**Comment by Luffy03** [comment]
Dear jsadu, the 'split_final.json' of nnUNet is generated by nnUNet preprocessing since we want to conduct fair comparisons with nnUNet. The data split json files of MSD tasks in Downstream are for **offline validation not for test** since we don't have the labels of test sets.
